environment/directories.rst

```diff
@@ -0,0 +1,84 @@
+######################
+Filesystem directories
+######################
+
+This page describes the filesystem directories available in the Notebook Aspect.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Directory
+     - Private
+     - Persistent
+     - Writeable
+   * - :ref:`filesystem-home`
+     - ✅
+     - ✅
+     - ✅
+   * - :ref:`filesystem-notebooks`
+     - ✅
+     - ✅
+     - ✅
+   * - :ref:`filesystem-datasets`
+     -
+     - ✅
+     -
+   * - :ref:`filesystem-project`
+     -
+     - ✅
+     - ✅
+   * - :ref:`filesystem-scratch`
+     -
+     -
+     - ✅
+
+.. _filesystem-home:
+
+$HOME
+=====
+
+Your home directory is just that.
+It’s backed by the same storage as your home directory on `lsst-dev`_ in the LSST Data Facility.
+In fact, your :file:`$HOME` directory in the Notebook Aspect *is* the :file:`$HOME/jhome/` directory on `lsst-dev <https://developer.lsst.io/services/lsst-dev.html>`_. 
```

Comment:
`$HOME/jhome/` is actually a symlink to `/lsst/jhome/$USER`.  We should double check with Michelle that these have the same policies as `$HOME`. (unless you already have)